### ISSUE TITLE
configuration.smk

### ISSUE THREAD

**Issue by JGASmits** [vanheeringen-lab/snakemake-workflows#253]
I just updated the pipeline, however when running it I ran into a issue with the configuration.smk file:

AttributeError in line 355 of /home/jsmits/snakemake-workflows/rules/configuration.smk:
'NoneType' object has no attribute 'update'
  File "/home/jsmits/snakemake-workflows/workflows/rna_seq/Snakefile", line 11, in <module>
  File "/home/jsmits/snakemake-workflows/rules/configuration.smk", line 355, in <module>

My first guess is that I should update my snakemake-workflows conda environment, since it doesn't know the .update command from the config file. Did the snakemake-workflows.yaml file get updated?

I'm running an update command right now, and will let you guys know if this fixes the issue.

Gr Jos.

**Comment by JGASmits** [comment]
Aaah, updating the snakemake-workflows env indeed seems to have fixed the issue. I'm not sure if/how you could prevent people from running into this problem in the future (since I cant think of a way to give a meaningfull error in snakemake). But then agian it also wouldnt really come up quit often.

**Comment by Maarten-vd-Sande** [comment]
In its current form this really sucks, there is not proper way to tell if people have the correct environment. I made a small change (see #255) that shouid at least catch it if Snakemake has been updated.

@siebrenf and I were discussing how we can circumvent these problems last week, and one of our solutions was to have a copy of the workflows + environment in one of our home folders on mb07, and have people use those instead of their own copies.
